packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1591,20 +1591,34 @@ class RenderPhysicalModel extends _RenderPhysicalModelBase<RRect> {
   void paint(PaintingContext context, Offset offset) {
     if (child != null) {
       _updateClip();
-      final RRect offsetClipRRect = _clip.shift(offset);
-      final Rect offsetBounds = offsetClipRRect.outerRect;
-      final Path offsetClipPath = new Path()..addRRect(offsetClipRRect);
+      final RRect offsetRRect = _clip.shift(offset);
+      final Rect offsetBounds = offsetRRect.outerRect;
+      final Path offsetRRectAsPath = new Path()..addRRect(offsetRRect);
+      bool paintShadows = true;
+      assert(() {
+        if (debugDisableShadows) {
+          context.canvas.drawRRect(
+            offsetRRect,
+            new Paint()
+              ..color = shadowColor
+              ..style = PaintingStyle.stroke
+              ..strokeWidth = elevation * 2.0,
```

Comment:
Why * 2?

Reply:
To make the difference more obvious. It was just a guess as to what would look good, I'm happy to change it to 1.0 if you like.

Reply:
That's what I was figuring.  2.0 sgtm